tests/testthat/test-use_readme.R

```diff
@@ -23,11 +23,16 @@ test_that("check_overwrite works", {
     check_overwrite(FALSE, golem_sys("utils/empty_readme.Rmd")),
     "README.Rmd already exists. Set `overwrite = TRUE` to overwrite."
   )
+  # Check if file exists
+  expect_true(file.exists(golem_sys("utils/empty_readme.Rmd")))
+  # Remove file via check_overwrite setting overwrite=TRUE
+  check_overwrite(TRUE, golem_sys("utils/empty_readme.Rmd"))
```

Comment:
Issue here is that the file will be removed from the installed package on the machine, so I'll rewrite this a bit :)